generateIndices.py

```diff
@@ -0,0 +1,81 @@
+#reads in files from features.npz
+# output training and testing indices
+import sys
+import re
+import math
+import numpy as np
+
+def generateIndex():
+
+    feature_file = sys.argv[1]
```

Comment:
I think move the ```sys.argv[1]``` reference into the ```if __name__ == "__main__"``` bit and pass ```feature_file``` into ```generateIndex()``` as a parameter? That way in another file we can do:
```
import generateIndex from generate_indices

i = generateIndex("path/to/file")
```